app/controllers/scans_controller.rb

```diff
@@ -27,20 +27,11 @@ def new_request_params
   end
 
   def redirect_to_success_with_token
-    redirect_to illiad_query(current_request)
+    redirect_to illiad_url
```

Comment:
If we don't need the user to do anything, why can't we automate this step? Is there something about the user credentials? Is there configuration we can do to allow the app to send proxy requests on their behalf? I know we've done some clever Apache configuration before to do this, and I'd guess we could do something similar here.